### PR TITLE
Joe/eccns

### PR DESCRIPTION
**Pull request by joelanders**
https://www.pivotaltracker.com/story/show/94882942

### CODE REVIEW

**Comment by joelanders** [comment]
Rebase, repush.

**Comment by joelanders** [comment]
rebase

File: app/importers/eccn_data.rb [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    doc[:eccn2] = row[3]
+    doc[:eccn3] = row[4]
+    doc[:eccn4] = row[5]
+
+    eccns = row[1..5].compact
+    urls = eccns.map { |eccn| eccn_to_url(eccn) }.uniq
+
+    doc[:url0] = urls[0] rescue nil
+    doc[:url1] = urls[1] rescue nil
+    doc[:url2] = urls[2] rescue nil
```
Comment (calumhalcrow) [inline app/importers/eccn_data.rb]: What type of error do you expect to suppress with the trailing `rescue`? As long as `urls` is an array, looking up a non-existent index won't fail; `nil` will be returned. I may be missing something, but I don't think these `rescue`s are necessary.

**Comment by calumhalcrow** [comment]
The importer isn't completely covered:

screen shot 2015-06-11 at 3 34 23 pm

**Comment by joelanders** [comment]
Fixed coverage.

**Comment by calumhalcrow** [comment]
ok, thanks @joelanders. Since Stuart has asked you to make some further changes, I won't merge this now.

**Pull request merged**